lib/plugins/sfDoctrinePlugin/lib/generator/sfDoctrineFormFilterGenerator.class.php

```diff
@@ -19,6 +19,17 @@
  */
 class sfDoctrineFormFilterGenerator extends sfDoctrineFormGenerator
 {
+
+    /**
+     * @var Doctrine_Table
+     */
+    public $table;
```

Comment:
You should delete them from here, because you have already added them in the `sfDoctrineFormGenerator` class.